.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Something is not working as expected
+labels:
+---
+
+**Before you start**
+Please take a look at the [FAQ](https://github.com/GoogleChrome/web-vitals-extension#faq) as well as the already opened issues! If nothing fits your problem, go ahead and fill out the following template:
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
Perhaps it would be useful to provide some examples? (e.g "what URLs/tabs did you have open when you noticed the issue?" tends to be common)